frontend/cotangles/src/App.tsx

```diff
@@ -1,29 +1,29 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
-function App() {
-  const [count, setCount] = useState(0);
+// Documentation: https://reactrouter.com/en/v6.3.0/getting-started/overview#navigation
+
+import Login from "./pages/Login";
+import Settings from "./pages/Settings";
+import MyCalendars from "./pages/MyCalendars";
+import Help from "./pages/Help";
+import Calendar from "./pages/Calendar";
 
+function App() {
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>count is {count}</button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">Click on the Vite and React logos to learn more</p>
+      <BrowserRouter>
+        <Routes>
+          {/* If not logged in already, redirect to login, otherwise redirect to my calendars.*/}
+          <Route path="/" element={<Login />}></Route>
+          <Route path="login" element={<Login />}></Route>
+          <Route path="my-calendars" element={<MyCalendars />}>
+            <Route path=":calendarId" element={<Calendar />} />
```

Comment:
also looks like this doesn't work as expected. I got it to work by putting my-calendars/:calendarid as a separate route but I feel like there's a nicer way. I imagine the nested routes is for the situation where the corresponding components are also nested, but that isn't the case here if im reading things correctly

Reply:
hmm im ngl i just copy pasted off the documentation but yeah strange that it wouldn't work here

Reply:
wait ill come back to this a bit later my 1531 presentation calls